pixeltable/env.py

```diff
@@ -68,6 +69,10 @@ def __init__(self):
         self._log_to_stdout = False
         self._module_log_level: Dict[str, int] = {}  # module name -> log level
 
+        # config
+        self._config_file: Optional[Path] = None
+        self._config: Optional[Dict[str, any]] = None
```

Comment:
It should be Any, not any.

(What does 'any' resolve to?)

Reply:
Oops. This was a typo. I'm surprised it runs! (re any vs. Any: I don't know. But now I'm curious.)